### PR TITLE
update IS version, use local mail, add sample apps

### PR DESCRIPTION
**Pull request by ghost**
Additionally add a mail server and update compose version to add more functionality and a modern example.

Local mail server allows for user signup example.

### CODE REVIEW

**Comment by vordimous** [comment]
This is my PR. I am consolidating my accounts and I should have closed before deleting the old account.

File: docker-compose/is/conf/identity-server/repository/conf/deployment.toml [hunk 1]
```diff
@@ -13,7 +13,9 @@ type = "read_write_ldap_unique_id"
 connection_url = "ldap://localhost:${Ports.EmbeddedLDAP.LDAPServerPort}"
 connection_name = "uid=admin,ou=system"
 connection_password = "admin"
-base_dn = "dc=wso2,dc=org"      #refers the base dn on which the user and group search bases will be generated
+base_dn = "dc=wso2,dc=org"
+#refers the base dn on which the user and group search bases will be generated
```
Comment (jyoakum) [inline docker-compose/is/conf/identity-server/repository/conf/deployment.toml:17]: I thought the typical convention was that if a comment wasn't on the same line then it was above the related code.

File: docker-compose/is/docker-compose.yml [hunk 2]
```diff
@@ -1,34 +1,53 @@
-version: '2.3'
+version: '3.7'
 services:
-  mysql:
```
Comment (jyoakum) [inline docker-compose/is/docker-compose.yml:3]: Wait a second. You can't remove the mysql DB from the mix while specifying that it be used in the deployment.toml (see the [database.identity_db] and [database.shared_db] sections).

Besides that, the mysql DB is what is making this docker-compose a production level product. It looks like this commit is not intended for MASTER but should be on a branch called "Sample-App-Playground" or something along those lines.

File: docker-compose/is/docker-compose.yml [hunk 3]
```diff
@@ -1,34 +1,53 @@
...
+  #   volumes:
+  #     - ./conf/mysql/scripts:/docker-entrypoint-initdb.d
+  #     - ./conf/mysql/conf/my.cnf:/etc/mysql/my.cnf
+  #   healthcheck:
+  #     test: ["CMD", "mysqladmin" ,"ping", "-uroot", "-proot"]
+  #     interval: 30s
+  #     timeout: 60s
+  #     retries: 5
 
+  # mailserver:
```
Comment (jyoakum) [inline docker-compose/is/docker-compose.yml:20]: The mail server settings were added to deployment.toml but the mailserver is commented out here.

File: docker-compose/is/dockerfiles/is/Dockerfile [hunk 4]
```diff
@@ -17,7 +17,7 @@
 # ------------------------------------------------------------------------
 
 # set base Docker image to WSO2 Identity Server Docker image with latest WSO2 Updates
-FROM docker.wso2.com/wso2is:5.11.0
+FROM docker.wso2.com/wso2is:5.11.0.5
```
Comment (jyoakum) [inline docker-compose/is/dockerfiles/is/Dockerfile:20]: There is a 5.11.0.19 that is available.

Comment (vordimous) [reply docker-compose/is/dockerfiles/is/Dockerfile]: updated

File: docker-compose/is/docker-compose.yml (hunk 2 above)
Comment (vordimous) [reply docker-compose/is/docker-compose.yml:3]: updated

**Comment by vordimous** [comment]
This was my PR from a throwaway account. I thought these would have closed automatically when I deleted the account. this PR can be closed

**Pull request closed without merging**